### ISSUE TITLE
[BUG] [ISSUE] Creating CNAME www.mydomain.com for mydomain.com on facileManager v2.3.3

### ISSUE THREAD

**Issue by kukinai** [WillyXJ/facileManager#390]
Hello!
My facileManager v2.3.3, bind-9.9.4-29.el7_2.4.x86_64
I've tryed create simple www. CNAME for my main domain:
A record                mydomain.com
CNAME record      www.mydomain.com
If I use "append domain" mark. I can't create CNAME record with empty Value (only mydomain.com)
If I DO NOT use "append domain" mark. I can't create CNAME record with dot at the end. 
Without dot (.) at the end, my bind DO NOT work. 

```
; Aliases
www.mydomain.com                300   IN   CNAME   mydomain.com.

```
```
[root@dc01-vm-ns01 ~]# dig www.mydomain.com @127.0.0.1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;www.nspk.com.                  IN      A

;; AUTHORITY SECTION:
mydomain.com.               28800   IN      SOA     ns01.mydomain.com. root.mydomain.com. 2017083102 7200 300 610000 28800

```
With dot at the end of CNAME record, all works perfectly.... But I can't validate it!
So I create A record www.mydomain.com with IP value, but it's no good :(

**Comment by WillyXJ** [comment]
Hi. Until better support is added, you can set 'www' as the record name, 'mydomain.com.' as the record value, and 'no' for append domain.

**Comment by WillyXJ** [comment]
Another option is to set 'www' as the record name, '@' as the record value, and 'no' for append domain.

**Comment by WillyXJ** [comment]
This is now fixed in [fmDNS 3.1](http://www.facilemanager.com/modules/fmdns/) and later.
